modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -597,14 +604,13 @@ public void onDisappeared(ClusterNode member) {
     }
 
     private void scheduleRemoveFromLogicalTopology(CmgRaftService raftService, ClusterNode node) {
-        // TODO: delay should be configurable, see https://issues.apache.org/jira/browse/IGNITE-16785
         scheduledExecutor.schedule(() -> {
             ClusterNode physicalTopologyNode = clusterService.topologyService().getByConsistentId(node.name());
 
             if (physicalTopologyNode == null || !physicalTopologyNode.id().equals(node.id())) {
                 raftService.removeFromCluster(Set.of(node));
             }
-        }, 0, TimeUnit.MILLISECONDS);
+        }, configuration.logicalTopologyRemovalDelayAfterDisappearance().value(), TimeUnit.MILLISECONDS);
```

Comment:
Configuration names this long might now be user-friendly. Some day we should revisit all names. Also, should this be a top-level configuration property, or maybe nested inside of some sub-configuration?

Anyway, name is too specific, user should not know what a disappearance. In Ignite 2 we called it a "failover timeout" if I remember correctly. We can discuss other options as well

Reply:
For now we just have 3 properties there, all of them are timeouts, one is for invoke timeout, other 2 are about CMG logic. It seems that it's too early to try to establish any substructure, at least it would be too hazy.

I agree, the name is too long. I failed to invent anything both meaningful and short, so I just adopt your suggestion about `failoverTimeout`.

Reply:
Thank you! I still hope that one day we will revisit all configuration names. And I mean all of them